### PR TITLE
Adding more information to Installing kubeadm step.

### PR DESCRIPTION
**Pull request by miteshskj**
Fixes https://github.com/kubernetes/website/issues/32880

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -70,9 +70,9 @@ Any commands under `kubeadm alpha` are, by definition, supported on an alpha lev
 
 ## Instructions
 
-### Installing kubeadm on your hosts
+### Preparing the hosts
 
-See ["Installing kubeadm"](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
+Install {{< glossary_tooltip term_id="container-runtime" text="container runtime" >}}  and kubeadm on all the hosts. For detailed instructions and other prerequisites, see ["Installing kubeadm"](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:75]:
```suggestion
Install a {{< glossary_tooltip term_id="container-runtime" text="container runtime" >}} and kubeadm
on all the hosts. For detailed instructions and other prerequisites, see ["Installing kubeadm"](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
```

- broke it down into 2 lines
- added missing `a` before `container runtime`
- removed extra space before `and kubeadm`

thanks for the PR, the change SGTM.

**Review by neolit123 — COMMENTED** [review]
/sig cluster-lifecycle
/triage accepted

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -70,9 +70,10 @@ Any commands under `kubeadm alpha` are, by definition, supported on an alpha lev
 
 ## Instructions
 
-### Installing kubeadm on your hosts
+### Preparing the hosts
 
-See ["Installing kubeadm"](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
+Install a {{< glossary_tooltip term_id="container-runtime" text="container runtime" >}} and kubeadm on all the hosts.
+For detailed instructions and other prerequisites, see ["Installing kubeadm"](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
```
Comment (tengqm) [inline content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:76]: I don't think the quotes are needed.

Comment (miteshskj) [reply content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md]: @tengqm  - removed them.

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Pull request merged**